waspc/cli/src/Wasp/Cli/Command/Start.hs

```diff
@@ -26,6 +26,8 @@ start = do
   waspRoot <- findWaspProjectRootDirFromCwd
```

Comment:
Hm, a thought -> could we use this Require mechanism to obtain the waspRoot easily?

Imagine that we could say that command requires to be in wasp project. That would be checked, but only that -> waspRoot would also be obtained.

I guess you would need to expand it a bit, so that we don't store just the fact that requirement is met, but also its "payload"/"value", but that shouldn't be hard.

I am not sure if we should do it though hm. Sometimes it is nice when commands are explicit about what they need in their arguments.

Reply:
I think that's an interesting idea! I talk about your idea here in one of my other comments, since its a solution for a couple other issues.

But I agree that its a bit nicer when its more explicit, and I'm not sure if we should leave this Requires thing for requirements that don't have data associated with them (like with waspRoot).